CONTRIBUTING.md

```diff
@@ -64,6 +64,7 @@ you will most likely use this for WSL), `homebrew` (macOS / Linux), and more:
 If you need one, you can find them on [AdoptOpenJDK](https://adoptopenjdk.net/).
 - `maven` (often package `maven`; can be found on
 [Apache's site](https://maven.apache.org/download.cgi) too).
+- `curl` (package `curl` everywhere);
```

Comment:
```suggestion
[Apache's site](https://maven.apache.org/download.cgi) too);
- `curl` (package `curl` everywhere).
```

Reply:
haha i didnt know that was a thing, ill commit that in a second